lab-shelly/README.md

```diff
@@ -0,0 +1,15 @@
+![cf](https://i.imgur.com/7v5ASc8.png) Lab 29 - ngAdventure
+======
+
+# Overview
+
+ngAdventure is a text-based adventure game that allows players to select from a dropdown form and navigate between rooms. The player stats and game history are displayed on the page.
+
+This lab builds on coursework in Angular JS, particularly components, services, custom directives, controllers, and dependency injection. Both controllers and services contain business logic; however, unlike controllers, services are reusable in other parts of an application and independent from the view. 
+
+## Installation
+* Clone this repository
```

Comment:
Nice job with the readme. Don't forget to include information on how to actually play the game as well.